hikari/impl/entity_factory.py

```diff
@@ -1375,10 +1375,9 @@ def deserialize_member(
             time.iso8601_datetime_string_to_datetime(raw_premium_since) if raw_premium_since is not None else None
         )
 
+        communication_disabled_until: typing.Optional[datetime.datetime] = None
         if raw_communication_disabled_until := payload.get("communication_disabled_until"):
             communication_disabled_until = time.iso8601_datetime_string_to_datetime(raw_communication_disabled_until)
```

Comment:
Double definition for no reason

Reply:
This is exactly how it is done in many other methods.